### ISSUE TITLE
bug: custom message doesn't apply

### ISSUE THREAD

**Issue by pakkographic** [favware/npm-deprecate#139]
```json
{
    "$schema": "https://raw.githubusercontent.com/favware/npm-deprecate/main/assets/npm-deprecate.schema.json",
    "name": "*dev*",
    "deprecateDistTag": false,
    "verbose": true,
    "package": [
      "touchguild"
    ],
    "message": "This version has been superseded by a new build, update."
  }
```

When deprecating, the custom message I set in .npm-deprecaterc.json doesn't work. It still deprecates with the default message.
Which is : "This version has been automatically deprecated by @favware/npm-deprecate. Please use a newer version."

**Comment by pakkographic** [comment]
Btw, I tried to create .npm-deprecaterc.json, and .npm-deprecaterc, but it still doesn't.

**Comment by pakkographic** [comment]
@favna

**Comment by favna** [comment]
> @favna

I saw the issue 6 days ago when you made it. I just haven't had time to address it. Comes with having a full time job and getting burned out from coding.

**Comment by pakkographic** [comment]
> > @favna
> 
> I saw the issue 6 days ago when you made it. I just haven't had time to address it. Comes with having a full time job and getting burned out from coding.

Sorry for this, since I have automated deprecation, it deprecates every time I push a commit with the wrong message, so it was somewhat important to me to fix this issue.

**Comment by pakkographic** [comment]
I still have no answer

**Comment by favna** [comment]
It should be fixed in v1.0.6. 

Anyway, I think you stress far too much about the message. From my experience, most people who use NPM don't read about a dependency being deprecated in the first place let alone the exact message for that deprecation.

**Comment by pakkographic** [comment]
Yeah but I'm a bit materialistic, thanks so much though.
